cloudformation.yaml

```diff
@@ -184,32 +208,38 @@ Resources:
                      "Next": "FailState"
                    }
                  ],
-                 "Default": "AddNewElasticsearchNode"
+                 "Default": "AddNode"
```

Comment:
disable rebalancing?

Reply:
That's included in the AddNode step - I've added a comment to lambda description to clarify this